src/Components/Publishing/Byline/Author.tsx

```diff
@@ -0,0 +1,66 @@
+import { Sans } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { getAuthorByline } from "../Constants"
+import { BylineLayout } from "../Typings"
+
+interface AuthorProps {
+  authors?: any
+  color?: string
+  layout?: BylineLayout
+  size?:
```

Comment:
@eessex Doesn't that size type live in palette already? Why not pull it from there?

Reply:
Oh meant to leave a todo here- is not exported, going to change in palette and replace here and other instances

Reply:
This has been updated to use Palette types